packages/components/psammead-story-promo/src/__snapshots__/index.test.jsx.snap

```diff
@@ -174,3 +181,274 @@ exports[`StoryPromo should render correctly 1`] = `
   </div>
 </div>
 `;
+
+exports[`StoryPromo with Media Indicator should render correctly 1`] = `
+.c7 {
+  -webkit-clip-path: inset(100%);
+  clip-path: inset(100%);
+  -webkit-clip: rect(1px,1px,1px,1px);
+  clip: rect(1px,1px,1px,1px);
+  height: 1px;
+  overflow: hidden;
+  position: absolute;
+  width: 1px;
+}
+
+.c3 {
+  padding: 0.5rem 0.25rem;
+  background-color: #FFFFFF;
+  display: inline-block;
+  font-family: ReithSans,Helvetica,Arial,sans-serif;
+  font-size: 0.75rem;
+  line-height: 1rem;
+  color: #222222;
+}
+
+.c4 {
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
```

Comment:
How does this look without flex? Mainly thinking old opera mobile and IE https://caniuse.com/#search=flex

Reply:
The flex in media indicator is just to make the vertical alignment a bit more even, looks nearly as nice without flex :)